RecoParticleFlow/PFTracking/python/mergedElectronSeeds_cfi.py

```diff
@@ -1,9 +1,15 @@
 import FWCore.ParameterSet.Config as cms
 
 
-
 electronMergedSeeds =cms.EDProducer("ElectronSeedMerger",
      EcalBasedSeeds = cms.InputTag("ecalDrivenElectronSeeds"),
      TkBasedSeeds  = cms.InputTag("trackerDrivenElectronSeeds:SeedsForGsf")
     )
 
+electronMergedSeedsFromMultiCl = electronMergedSeeds.clone()
```

Comment:
same comment as before about cfi files and no need for phase2 modifier applies here